mappings/net/minecraft/world/gen/DensityFunction.mapping

```diff
@@ -0,0 +1,46 @@
+CLASS net/minecraft/unmapped/C_kzwapyud net/minecraft/world/gen/DensityFunction
+	FIELD f_hewnhqgm HOLDER_HELPER_CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_usxhasin DIRECT_CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_xvwcxrho CODEC Lcom/mojang/serialization/Codec;
+	METHOD m_apksyljd squeeze ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_gmrgdcrk quarterNegative ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_hfnqnnds minValue ()D
+	METHOD m_idbpaztj halfNegative ()Lnet/minecraft/unmapped/C_kzwapyud;
```

Comment:
What is a half negative?

Reply:
It is a type provided my mojang. One that I can't change. It basically multiplies it by a half
```java
if (d > 0.0) {
    yield d;
}
yield d * 0.5;
```